apps/web/src/features/organizations/components/MembersList/RemoveMemberModal.tsx

```diff
@@ -0,0 +1,50 @@
+import ModalDialog from '@/components/common/ModalDialog'
+import { DialogContent, DialogActions, Button, Typography } from '@mui/material'
+import type { UserOrganizationUser } from '@safe-global/store/gateway/AUTO_GENERATED/organizations'
+import { useUserOrganizationsRemoveUserV1Mutation } from '@safe-global/store/gateway/AUTO_GENERATED/organizations'
+import { useCurrentOrgId } from '@/features/organizations/hooks/useCurrentOrgId'
+import { useState } from 'react'
+
+const RemoveMemberDialog = ({ member, handleClose }: { member: UserOrganizationUser; handleClose: () => void }) => {
+  const orgId = useCurrentOrgId()
+  const [deleteMember] = useUserOrganizationsRemoveUserV1Mutation()
+  const [errorMessage, setErrorMessage] = useState<string>('')
+
+  const handleConfirm = async () => {
+    setErrorMessage('')
+    try {
+      const { error } = await deleteMember({ orgId: Number(orgId), userId: member.id })
+
+      if (error) {
+        throw error
+      }
+    } catch (e) {
+      setErrorMessage('An unexpected error occurred while removing the member.')
+    }
```

Comment:
The dialog doesn't close when deleting a member. I would suggest calling an onClose function in a finally block.